assets/src/edit-story/components/form/color/colorInput.js

```diff
@@ -121,6 +121,9 @@ const StyledSwatch = styled(Swatch)`
   ${focusStyle};
 `;
 
+const loadReactColor = () =>
```

Comment:
All these dynamic imports might need a `.catch()` in case webpack fails to load them for whatever reason, not sure.

Reply:
It looks like using `ErrorBoundary` seems to be the recommended way for catching issues with it. We already have those per area, not sure if it would make sense to add more. Thoughts?